pyiron/atomistics/structure/atoms.py

```diff
@@ -1103,6 +1154,55 @@ def center_coordinates_in_unit_cell(self, origin=0, eps=1e-4):
             )
         return self
 
+    def band_structure_calculation(self,
```

Comment:
I guess it's not easy to understand `band_structure_calculation` is a function that activates a band structure calculation. Maybe something like `set_band_structure_calculation` or `activate_band_structure_calculation`. To quote Liam: functions should start with a verb.

Reply:
Sure, I also thought about the name. `set` sounds like setting a variable. I would suggest `create_structure_for_line_mode` or `create_line_mode_structrue`.

Reply:
I just realized that it's in `atoms.py`, but shouldn't it rather be inside `GenericDFTJob`? I mean in `pyiron.dft.job.generic`

Reply:
I would keep it in `atoms.py`. Then the user can create a structure and check the symmetries and paths before creating a job.

Reply:
new function name